.github/workflows/submit.yml

```diff
@@ -400,24 +377,52 @@ jobs:
           path: ~/linux-x64${{ matrix.artifact }}_testsupport_${{ env.logsuffix }}.zip
         continue-on-error: true
 
-  linux_aarch64_build:
-    name: Linux aarch64
+  linux_x64_hotspot_only:
```

Comment:
This is a weird name for a job that also cross-compiles non-x64 versions. Maybe `linux_aux_builds` or some such?

Reply:
Yeah this can probably be improved, how about the updated version?